pkg/api/mock/matcher_object_deleted.go

```diff
@@ -0,0 +1,30 @@
+package mock
+
+import (
+	"fmt"
+
+	gomegaTypes "github.com/onsi/gomega/types"
+)
+
+// ObjectDeleted checks if mockAPIObject was deleted
+func ObjectDeleted(tags ...string) gomegaTypes.GomegaMatcher {
+	return &objectDeleted{}
+}
+
+type objectDeleted struct{}
+
+func (m *objectDeleted) Match(actual interface{}) (bool, error) {
+	obj, ok := actual.(*mockAPIObject)
+	if !ok {
+		return false, fmt.Errorf("ObjectDeleted matcher expects an *mockAPIObject")
```

Comment:
first: why not `errors.New()` instead of `fmt.Errorf()`

second: please don't ever return errors created on the fly ^^'